addons/web/static/src/actions/action_service.js

```diff
@@ -507,7 +508,7 @@ function makeActionManager(env) {
           });
           mode = "new";
         }
-        resolve();
+        resolve(controller.action.id);
```

Comment:
This resolves the controller, so resolving with just the actionId, seems out of bounce from the outside

Reply:
it depends, it's the id of the current action (considering this is the **action** service, it kinda makes sense)